charts/sn-platform/templates/bookkeeper/bookkeeper-cluster.yaml

```diff
@@ -158,34 +158,31 @@ spec:
         {{- include "pulsar.bookkeeper.ledgers.storage.class" . | nindent 8 }}
     reclaimPolicy: Delete
   {{- end }}
-  {{- if .Values.bookkeeper.custom }}
```

Comment:
With this gone, I see .Values.bookkeeper.custom no longer used.

If the intent is to put all custom config in configData, we could remove [.custom from the values file](https://github.com/streamnative/charts/blob/master/charts/sn-platform/values.yaml#L879).